src/utils/md.ts

```diff
@@ -4,13 +4,13 @@ import * as path from 'path';
 
 function convertTitle(md: string): string {
   return md.replace(/^#\s+(.*)/gm, (match, title: string) => {
-    return title.toLocaleUpperCase() + EOL + Array(title.length + 1).join('=');
+    return '\x1b[1m' + title.toLocaleUpperCase() + EOL + Array(title.length + 1).join('=') + '\x1b[0m';
```

Comment:
Instead of manually handling escape codes, let's use chalk which we already use in the CLI and which handles all of this for you, making the code more readable and easier to maintain

Reply:
Thank you @waldekmastykarz! Changed the use to chalk. Just wanted to let you know that chalk eventually does insert the escape characters. But it uses a slighly different format than the one from the first commit here.

That's why you'll see a change to the regex for hyperlink because he falsly think the subtitle after the options section is a hyperlink because of the escape characters